platform/npm/test.js

```diff
@@ -32,7 +32,7 @@ import { promisify } from 'util';
 /******************************************************************************/
 
 async function spawnMocha() {
-    await promisify(spawn)('mocha', [ '--experimental-vm-modules', '--no-warnings', 'tests' ], { stdio: [ 'inherit', 'inherit', 'inherit' ] });
+    await promisify(spawn)('mocha', [ '--experimental-vm-modules', '--no-warnings', 'tests', '--reporter', 'progress' ], { stdio: [ 'inherit', 'inherit', 'inherit' ] });
```

Comment:
@gorhill we need a different reporter because otherwise there's too much output. This one will only show something if a test fails.